src/tests/ie_test_utils/common_test_utils/data_utils.hpp

```diff
@@ -85,7 +89,7 @@ make_reshape_view(const InferenceEngine::Blob::Ptr &blob, InferenceEngine::SizeV
  * @param blob tensor to fill in
  * @param val value to set into each element
  */
-void fill_data_const(InferenceEngine::Blob::Ptr &blob, float val);
+void fill_data_const(const ov::Tensor& tensor, float val);
```

Comment:
We can avoid of using this method:
```
    std::vector<float> values = {-1, -2.05, -3.65, 0, 5, 7};
    auto const_node = std::make_shared<ov::op::v0::Constant>(ov::element::Type_t::f32, ov::Shape({2, 3}), values);
```

Look like this method is extra and not needed more

Reply:
fill_data_const is really extra method. But I changed it to CommonTestUtils::fill_data_with_broadcast because it is more basic